Changelog.md

```diff
@@ -7,6 +7,13 @@ This project does _not_ adhere to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html) but contrary to it
 every new version is a new major version.
 
+## 121 - UNRELEASED
+
+### Changed
+
+-   Analytic events names of prod and dev are now distinct with a prefix of
+    `Prod:` or`Dev:`
```

Comment:
**Analytics** is the brand name used by Google for their Telemetry. I suggest that we try to refer to it as generically as possible, and therefore use Telemetry events, instead of Analytics events. The Microsoft equivalent to Google Analytics, I believe is called Azure Monitor Insights.

Suggestion:
```md
Telemetry events have distrinct prefixes `Prod:` and `Dev:`, where `Prod`: refers to all the sources except the `local` source in  nRF Connect for Desktop.`
```